File: variables.js
// Variables 
/* 
Definir Variables

var, let y const

Data Types:

String
Number
Boolean
Undefined

Object:
    Null
    Literal Object
    Array

function

*/

let nombre = "";
let apellido = '';
let direccion = ` texto 
sadsd
asdasd
asd

`;

let nombreCompleto = `${nombre} ${apellido}`;

let edad = 30; // number

let precio = 10.50; // number

let activo = false;

let open = true;

let postalCode; // undefined
let piso = null;


let persona = { // object
    nombre: 'John',
    apellido: 'Doe'
}

persona.nombre
persona["nombre"]

let nombres = ["Hugo", "Paco", "Luis"]
nombres[1]


function sumar(a, b){ // function
    return a + b;
}

File: variables.py
# Variables 
""" 

Definir variables 

(no tiene palabras reservadas para ello)


Data Types


String
Int
Float
Boolean

None

Dict

Array: 
    List
    Tuple
    Set

func

"""


nombre = ""
apellido = ''
direccion = '''

asdasd
asdasdad
asdasd
asd

'''

nombre_completo = f"{nombre} {apellido}"

edad = 30 # int
precio = 10.50 # float

activo = True

open = False

postal_code = None
piso = None

persona = { # dict 
    "nombre": "John",
    "apellido": "Doe"
}

persona["apellido"]

# Array

# List
nombres = ["Hugo", "Paco", "Luis"]
nombres[1]

# Tuple
estatus = ("activo", "inactivo", "cancelado", "aprobado")
estatus[3]

# Set
frutas = { "Pera", "Manzana", "Uva", "Mandarina", "Banana"}


def sumar(a, b):
    return a + b